tensornetwork/backends/numpy/numpy_backend_test.py

```diff
@@ -163,6 +165,13 @@ def test_randn(dtype):
   assert a.shape == (4, 4)
 
 
+@pytest.mark.parametrize("dtype", [np.complex64, np.complex128])
+def test_randn_non_zero_imag(dtype):
+  backend = numpy_backend.NumPyBackend(dtype=dtype)
```

Comment:
How are you checking that this is actually a complex value?

Reply:
you mean `a`? There is another test case where I check that the backend gives back the expected types
(`test_randn_dtype_2`) on line 229

Reply:
(this test and some of the others are actually wrong in the current master)

Reply:
Please fix them. Thanks!

Reply:
They are fixed in #253

Reply:
I have merged #253 already on my fork into this PR (sorry about that!) because I needed some of the stuff there here. Once #253 is merged I'll fetch the upstream/master and merge it into this PR to fix the conflicts